pom.xml

```diff
@@ -10,7 +10,7 @@
 
     <groupId>com.facebook.presto.ranger</groupId>
     <artifactId>ranger-apache</artifactId>
-    <version>2.1.0-4-SNAPSHOT</version>
+    <version>2.5.0</version>
```

Comment:
The version here should only be updated by the maven-release plugin

Reply:
Noted, just testing locally.